platform/ios/src/MGLMapView.h

```diff
@@ -1308,10 +1319,13 @@ MGL_EXPORT
  view’s frame. Otherwise, those properties are inset, excluding part of the
  frame from the viewport. For instance, if the only the top edge is inset, the
  map center is effectively shifted downward.
-
+ 
  When the map view’s superview is an instance of `UIViewController` whose
  `automaticallyAdjustsScrollViewInsets` property is `YES`, the value of this
  property may be overridden at any time.
+ 
+ The usage of `automaticallyAdjustsScrollViewInsets` it is been deprecated
```

Comment:
Nit: Split this into two sentences and replace “it is been” with “has been”. (×3)